07week/dom-practice/js/script.js

```diff
@@ -3,3 +3,71 @@
 document.addEventListener("DOMContentLoaded", function(event) {
   // You code here
 });
+
+function sayHi(){

```

Comment:
The delete item from cart button was optional, but i was expecting a clear cart button